src/elona/lua_env/mod_manager.cpp

```diff
@@ -271,13 +261,38 @@ local function clear(t)
 end
 if Store then
     clear(Store.map)
+end
+)",
+            mod->env);
+    }
+}
+
+void ModManager::clear_global_stores()
+{
+    for (auto&& pair : this->enabled_mods())
+    {
+        auto& mod = pair.second;
+        lua_->get_state()->safe_script(
+            R"(
+local function clear(t)
+    for key, _ in pairs(t) do
+        t[key] = nil
+    end
+end
+if Store then
     clear(Store.global)
```

Comment:
Doesn’t this work?
```lua
Store.global = {}
```

Reply:
Yes, it should. I don't quite remember why I wrote it this way.

Reply:
Actually I remember now. There is protection on `Store` to prevent it from being reassigned, so it will always be a table. However I think setting a plain value could also work and prevent confusion. I'll try changing it.

Reply:
Ah, understand.